_posts/2019-04-14-ferryman-post.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Our First Game Jam!"
+date:   2019-04-14 14:55:00 -0500
+categories: blog
+author: Aaron
+author_twitter: shamshirz
+image: /assets/ferryman-glass.png
+---
+_48 hour sprint to Unity greatness_
+[![Ferryman image]({{ "/assets/ferryman-glass.png" | relative_url }}){:class="excerpt-image"}]({{ site.baseurl }}{% post_url 2019-04-10-ferryman %})
+<!-- Ends the excerpt text, it includes the image -->
+
+
+How it works.
+
+* Start Friday.
+* Build a game that is loosely based on the theme of "Spirits".
+* Stop working on Sunday.
+
+This was the first time we've joined a challenge like this, so we decided to increase the difficulty by making it our first published game made in Unity. We broke up into 3 roles for the most part; Aaron on art; Sam building the C# game logic; RJ tying everything together in Unity. We each learned a lot about our roles, and how different they are from some of the other games we made.
+
+
+## Unity
+
+The built in collaboration tool was surprisingly great! We are accustomed [Git](https://github.com/SylverStudios) at work and for personal projects, but we tried the Unity built in collaboration tool and ended up sticking with it. There were a couple bumps along the way, but we highly recommend trying it out.
```

Comment:
built-in *

Reply:
This paragraph may work better if the conclusion (tool great!) comes after the problem statement (not used to it, new thing). Perhaps: "We started out worried about how we would collaborate on the game, and in such a rapid-fire way that the time constraints demanded. We are all accustomed to Git at work and for personal projects, but since Unity controls so much of the entire development experience, including the translation from what you see in the GUI to what gets stored in which files, Git doesn't seem to be a great tool for it. Luckily, Unity comes with it's own built-in collaboration tool that has its own push, pull, conflict resolution, and cloud storage solutions. We tried it out and, voila! It worked seamlessly, and allowed for us to hit a very high collaborative velocity."

Reply:
😮 🙏 very nice, swapperino. Possibly a little long, will do a total word count for the post at the end